src/applications/mhv/secure-messaging/tests/e2e/pages/PatientMessageDraftsPage.js

```diff
@@ -163,6 +163,14 @@ class PatientMessageDraftsPage {
     cy.get('[data-testid="delete-draft-button"]').click({ force: true });
   };
 
+  sendDraftMessage = draftMessage => {
+    cy.intercept('POST', '/my_health/v1/messaging/messages', draftMessage).as(
```

Comment:
I would suggest we should start looking into storing these path string values in a dedicated constants file and pull values from there.

Reply:
most of values amended by variables using `mhv/secure-messaging/util/constants.js`.
Please review and in case of approval this pattern will be gradually applied to all existing tests and implemented to further ones

Reply:
see comment above